assets/dev/js/editor/components/ai/pages/connect/index.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useRef } from 'react';
+import { Stack, Button, Typography, Link } from '@elementor/ui';
+import { AIIcon } from '@elementor/icons';
+
+const Connect = ( { connectUrl, onSuccess } ) => {
+	const approveButtonRef = useRef();
+
+	useEffect( () => {
+		jQuery( approveButtonRef.current ).elementorConnect( {
+			success: ( _, data ) => {
+				elementorCommon.config.library_connect.is_connected = true;
+				elementorCommon.config.library_connect.current_access_level = data.accessLevel;
```

Comment:
is it ok that a React component of some module changes that global variable? that's how it works? doesn't the jQuery thing takes care of that?